index.js

```diff
@@ -88,9 +90,7 @@ for (const platform of platforms) {
                 platformConfig.features[key].exceptions = platformConfig.features[key].exceptions.concat(platformOverride.features[key].exceptions)
             }
         }
-        if (isFeatureDisabled(platformConfig.features[key])) {
-            // If feature isn't enabled for platform remove.
-            // delete platformConfig.features[key]
+        if (isFeatureMissingState(platformConfig.features[key])) {
```

Comment:
Sorry, I'm still not the best person to review, but I was wondering - why would feature miss a state in the first place? Why don't we enforce state to always be there via JSON schema? This way we will force folks to directly declare it and don't make any assumptions.

Reply:
A feature might only be enabled on one platform.
The previous approach was to delete these keys but there's at least an issue in the extension currently doing this. In the new year we should be good to delete again.